modules/ossm-rn-known-issues.adoc

```diff
@@ -32,39 +32,32 @@ These are the known issues in {ProductName}:
++
+For example, if you create a namespace called 'akube-a' and add it to the Service Mesh member roll, then the Kiali UI does not display the namespace. For defined exclusion patterns, the software excludes namespaces that start with or contain the pattern.
++
+The workaround is to change the Kiali Custom Resource setting so it prefixes the setting with a carat (^). For example:
+
+  api:
+    namespaces:
+      exclude:
+      - "^istio-operator"
+      - "^kube-.*"
+      - "^openshift.*"
+      - "^ibm.*"
+      - "^kiali-operator"
+
 * link:https://issues.redhat.com/browse/OSSM-285[OSSM-285] When trying to access the Kiali console, receive the following error message "Error trying to get OAuth Metadata". The workaround is to restart the Kiali pod.
 
-* link:https://issues.redhat.com/browse/MAISTRA-2692[MAISTRA-2692] With Mixer removed, custom metrics that have been defined in {ProductShortName} 2.0.x cannot be used in 2.1. Custom metrics can be configured using `EnvoyFilter`. Red Hat is unable to support `EnvoyFilter` configuration except where explicitly documented. This is due to tight coupling with the underlying Envoy APIs, meaning that backward compatibility cannot be maintained.
+* link:https://issues.redhat.com/browse/MAISTRA-2735[MAISTRA-2735] The resources that the Service Mesh Operator deletes when reconciling the SMCP have changed. Previously, the Operator deleted a resource with the following labels:
+
+** `maistra.io/owner`
+** `app.kubernetes.io/version`
 
-* link:https://issues.jboss.org/browse/MAISTRA-2687[MAISTRA-2687] {ProductName} 2.1 federation gateway does not send the full certificate chain when using external certificates. The {ProductShortName} federation egress gateway only sends the client certificate. Because the federation ingress gateway only knows about the root certificate, it cannot verify the client certificate unless you add the root certificate to the federation import `ConfigMap`.
-+
-. To provide both the root certificate and CA certificate when setting the federation import `ConfigMap`:
-+
-[source,yaml]
-----
-  apiVersion: v1
-  kind: ConfigMap
-  metadata:
-    name: mesh1-ca-root-cert
-    namespace: mesh2-system
-  data:
-    root-cert.pem: |-
-      {{MESH1_CERT}}
-----
-+
-. Assign the certificate values to the mesh variable:
-+
-[source,terminal]
-----
-$ MESH1_CERT=$(cat cacerts/root-cert.pem cacerts/ca-cert.pem | sed ':a;N;$!ba;s/\n/\\\n    /g')
-----
-+
-. Insert the certificate information into `ConfigMap` and apply the change:
 +
-[source,terminal]
-----
-$ sed "s:{{MESH1_CERT}}:$MESH1_CERT:g" import/configmap.yaml | oc apply -f -
-----
+Now, the Operator ignores resources that don't also include the `app.kubernetes.io/managed-by=maistra-istio-operator` label. If you create your own resources, you should not add the `app.kubernetes.io/managed-by=maistra-istio-operator` label to them.
```

Comment:
Oh, here's part of what I was looking for above.  I'll still update my PR to make sure we're not sending mixed messagse to customers.